ANSWERS.md

```diff
@@ -2,7 +2,23 @@
 
 
 1. In Jest, what are the differences between beforeAll, afterAll, beforeEach, and afterEach? When do they run? What are they used for?
+    

```

Comment:
Glad you took the time to answer this questions. They come in handy on interviews